src/backend/src/services/projects.services.ts

```diff
@@ -811,4 +823,31 @@ export default class ProjectsService {
 
     return newMaterialType;
   }
+
+  /**
+   * Deletes a material type based on the given Id
+   * @param submitter the user who is deleting the material type
+   * @param materialTypeId the Id of the material type being deleted
+   * @returns the deleted material type
+   */
+  static async deleteMaterialType(materialTypeId: string, submitter: User): Promise<Material_Type> {
+    if (!isHead(submitter.role) && !isAdmin(submitter.role)) {
+      throw new AccessDeniedException('Only an admin or head can delete a material type');
+    }
+    const materialType = await prisma.material_Type.findUnique({
+      where: {
+        name: materialTypeId
+      }
+    });
+
+    if (!materialType) throw new HttpException(400, `This material type does not exist`);
```

Comment:
If something does not exist, it would be good to use the `NotFoundException`. There are many examples in the service file. Here is one: `if (!team) throw new NotFoundException('Team', teamId);`